concent_api/core/views.py

```diff
@@ -271,6 +288,72 @@ def handle_send_force_get_task_result(client_message: message.concents.ForceGetT
         )
 
 
+def handle_send_force_subtask_results(client_message: message.concents.ForceSubtaskResults):
+    assert client_message.TYPE in message.registered_message_types
+
+    current_time = int(datetime.datetime.now().timestamp())
+
+    if Message.objects.filter(
+        type    = client_message.TYPE,
+        task_id = client_message.ack_report_computed_task.task_to_compute.compute_task_def['task_id']
+    ).exists():
+        return message.concents.ServiceRefused(
+            timestamp   = client_message.timestamp,
```

Comment:
We can't pass timestamp to message like this, please take a look on https://github.com/golemfactory/concent/pull/89/files, there are some ways to work around this.

Reply:
We can in golem-messages <= 1.6.0, which is currently on master. Should I change it right now, expecting that I will rebase this branch on other that handle newer version of that library?

Reply:
In this case yes, we will be updating because we need messages from > 1.6.0.